src/components/Pages/addOrder.css

```diff
@@ -0,0 +1,22 @@
+.white-text {
+  color: #ffffff;
+}
+
+.addmember {
+  margin-left: 145px;
+  margin-top: 48px;
+}
+
+#orderform {
+  background: #0c0e0f;
+  color: #ffffff;
+}
+#order_h1 {
+  font-weight: 600;
+  font-size: 48px;
+  line-height: 58px;
+}
+.submit_btn {
+  background: #75bd00;
+  border-radius: 7px;
+}
```

Comment:
* Currently none of this CSS selectors are referring to anything. The .class selectors should refer to classNames and the #class selectors refer to id attributes
* Font-size should use rem sizing. Rem refers to the font-size set in the root element (which is in index.css), so 1rem = 16px and so on. Just makes things more consistent
* Spacing (padding, margin, width and height) should also refer to rem sizing in most cases too
* In the root element Khang set up some global styles for the colours used throughout our webpage. So rather then using "color: #ffffff" you should use "color: var(--clr-white)"